lib/process-services/src/lib/form/services/process-content.service.ts

```diff
@@ -183,6 +183,19 @@ export class ProcessContentService {
             .pipe(catchError((err) => this.handleError(err)));
     }
 
+    /**
+     * Lists processes and tasks on workflow started with provided document
+     *
+     * @param sourceId - id of the document that workflow or task has been started with
+     * @param source - source of the document that workflow or task has been started with
+     * @param size - size of the entries to get
+     * @param page - page number
+     * @return Promise<ResultListDataRepresentationRelatedProcessTask>
+     */
+    getProcessesAndTasksOnContent(sourceId: string, source: string, size?: number, page?: number): Observable<ResultListDataRepresentationRelatedProcessTask> {
```

Comment:
Should we write some unit tests as we have for other functions?

Reply:
This method just calls an api from JS-API and since Denys raised an idea to gradually move those to JS-APi maybe we can skip unit testing it for now? The tests would be very simple and it not really that much to check here. What do you think @AleksanderSklorz?